IS0401Test.py

```diff
@@ -570,34 +570,17 @@ def test_11_01(self, test):
 
     def test_12(self, test):
         """Node advertises a Node type mDNS announcement with no ver_* TXT records
-        in the presence of a Registration API"""
+        in the presence of a Registration API (v1.0, v1.1 and v1.2)"""
 
         if not ENABLE_DNS_SD:
             return test.DISABLED("This test cannot be performed when ENABLE_DNS_SD is False")
 
         api = self.apis[NODE_API_KEY]
 
-        registry_info = self._registry_mdns_info(self.primary_registry.get_data().port, 0)
-
-        # Reset the registry to clear previous data, although we won't be checking it
-        self.primary_registry.reset()
-        self.primary_registry.enable()
-
-        if DNS_SD_MODE == "multicast":
-            # Advertise a registry at pri 0 and allow the Node to do a basic registration
-            self.zc.register_service(registry_info)
-
-        # Wait for n seconds after advertising the service for the first POST from a Node
-        self.primary_registry.wait_for_registration(DNS_SD_ADVERT_TIMEOUT)
-
-        ServiceBrowser(self.zc, "_nmos-node._tcp.local.", self.zc_listener)
-        time.sleep(DNS_SD_BROWSE_TIMEOUT)
-        node_list = self.zc_listener.get_service_list()
+        if self.is04_utils.compare_api_version(api["version"], "v1.2") == 1:

```

Comment:
Just a suggestion as I find it less confusing if we always have zero on the right hand side of the comparison.

```suggestion
        if self.is04_utils.compare_api_version(api["version"], "v1.3") < 0:
```

Reply:
I do need to check this actually does what I think it does...